swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -195,6 +195,76 @@ version = "0.1.0"
 include = ["documents", "sample.png"]
 ```
 
+### The `keywords` field
+
+The keywords field is used to specify a list of short phrases that describe the package. Keywords are helpful for users to discover the package in the Ballerina Central using the built-in search. These keywords are also listed under the `Keywords` section in the Ballerina Central API documentation.
+
+The `keywords` field is optional and accepts a string array.
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+keywords = ["service", "edi", "manufacturing"]
+```
+
+### The `authors` field
+
+The `authors` field is used to specify the authors who contributed to the package. This information will be listed under the `Contributors` section in the Ballerina Central API documentation. 
+
+The `authors` field is optional and accepts a string array.
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+authors = ["John Doe", "Jane Doe"]
+```
+
+### The `repository` field
+
+The `repository` field is used to specify the URL of the repository where the source code of the package is hosted. This will be listed as `Source Repository` in the Ballerina Central API documentation.
+
+The `repository` field is optional and accepts a string.
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+repository = "https://github.com/john-doe/module-winery"
+```
+
+### The `license` field
+
+The `license` field is used to specify the licenses under which the package is distributed. This will be listed as `License` under the `Metadata` section in the Ballerina Central API documentation.
+
+The `license` field is optional and accepts a string array.
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+license = ["Apache-2.0"]
+```
+
+### The `template` field
```

Comment:
Let's remove this. We haven't announced this yet.

Reply:
Fixed with 407ba4be27d26bd90a0bc1938b40f615f289a1ec